terraform/india/development/main.tf

```diff
@@ -69,15 +69,39 @@ module "s3-nwp-bucket" {
   lifecycled_prefixes = ["ecmwf/data", "ecmwf/raw"]
 }
 
-/*
+
+
+resource "aws_iam_policy" "iam_policy_ecmwf_live_s3_read" {
```

Comment:
can we do this, I thought we would have to pass AWS KEY and AWS Secret for that user

Reply:
I'm not sure - I think it might work, as you just created a user with a policy on it that would be what was passed anyway, so this isn't miles away from that setup?

Reply:
If we enable ACLs on the bucket and remove "Bucket Owner Enforced" I think we can share the bucket between different accounts - https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-walkthroughs-managing-access-example2.html

But it might just be easier to pass credentials through! I don't mind which we do.

Reply:
Ill let you choose. Simpliest is often best